extensions/pdh/PerformanceDataMonitor.h

```diff
@@ -75,7 +78,9 @@ class PerformanceDataMonitor : public core::Processor {
   void addCustomPDHCountersFromProperty(const std::string& custom_pdh_counters);
 
   OutputFormat output_format_;
+  bool pretty_output_;
```

Comment:
This seems to be uninitialized.

Reply:
good catch, fixed in [26802f7](https://github.com/martinzink/nifi-minifi-cpp/commit/26802f77ef539757fcedbffea0f596b2ef0cc828)